pkg/fortify/fortify_test.go

```diff
@@ -39,7 +39,7 @@ func spinUpServer(f func(http.ResponseWriter, *http.Request)) (*SystemInstance,
 
 func TestCreateTransportConfig(t *testing.T) {
 	t.Run("Valid URL", func(t *testing.T) {
-		config := createTransportConfig("http://some.fortify.host.com/ssc", "api/v2")
+		config := createTransportConfig("http://some.fortify.host.com/ssc/", "/api/v2/")
```

Comment:
should we have more test cases for the range of "expected" inputs? I mean, without trailing slash will also work, right? please verify in a test-case.